integration-tests/src/polkadot_test_net.rs

```diff
@@ -179,6 +179,11 @@ pub fn hydra_ext() -> sp_io::TestExternalities {
 			(b"DAI".to_vec(), 1_000u128),
 			(b"USDC".to_vec(), 1_000u128),
 		],
+		asset_ids: vec![
+			(b"LRNA".to_vec(), 1_000u128, LRNA),
+			(b"DAI".to_vec(), 1_000u128, DAI),
+			(b"USDC".to_vec(), 1_000u128, DOT),
+		],
```

Comment:
This will add the same assets twice which is probably not what you want.
Maybe I should have set up the genesis config differently :thinking: (For one asset you basically want to add it to *either* `asset_names` *or* `asset_ids`.)

Reply:
i was wondering about that too. I was thinking just to have one and you have to provide asset id.